.pre-commit-config.yaml

```diff
@@ -16,7 +16,7 @@ repos:
     - id: ruff-format
 
 -   repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: v18.1.1
+    rev: v18.1.3
```

Comment:
would be nice to have in a separate commit in case of rollbacks.

Reply:
Sorry, left over. Will rm.